### PR TITLE
JC-2211 Highlight button localisation

### PR DESCRIPTION
**Pull request by RuslanSh**
For localisation of highlight button there was used Spring message
localisation.

### CODE REVIEW

File: jcommune-view/jcommune-web-view/src/main/resources/org/jtalks/jcommune/web/view/messages_ru.properties [hunk 1]
```diff
@@ -62,6 +62,7 @@ label.answer.italic=\u041A\u0443\u0440\u0441\u0438\u0432 (Ctrl+I)
 label.answer.underline=\u041F\u043E\u0434\u0447\u0435\u0440\u043A\u043D\u0443\u0442\u044B\u0439 (Ctrl+U)
 label.answer.striked=\u0417\u0430\u0447\u0435\u0440\u043A\u043D\u0443\u0442\u044B\u0439 (Ctrl+S)
 label.answer.highlight=\u041F\u043E\u0434\u0441\u0432\u0435\u0442\u043A\u0430
+label.answer.highlight.button=\u041f\u043e\u0434\u0441\u0432\u0435\u0442\u043a\u0430 \u0442\u0435\u043a\u0441\u0442\u0430
```
Comment (standpoint) [inline jcommune-view/jcommune-web-view/src/main/resources/org/jtalks/jcommune/web/view/messages_ru.properties]: @RuslanSh , could you please also correct the tooltips for the highlight button (in all languages): `label.answer.highlight`

Comment (RuslanSh) [inline jcommune-view/jcommune-web-view/src/main/resources/org/jtalks/jcommune/web/view/messages_ru.properties]: Yes, of course. Please, let me know what is wrong with it.

**Comment by standpoint** [comment]
there are tooltips in russian - "Подсветка", in ukrainian - "Підсвітка" now.
According to the rest of the buttons and their tooltips, these ones must be as button names: "Подсветка текста" / "Підсвічування тексту"

**Comment by RuslanSh** [comment]
@standpoint Done!

**Pull request merged**